templates/patterns/collapse/pattern-collapse.html.twig

```diff
@@ -1,18 +1,34 @@
 {% set id = 'collapse-' ~ random(1000) %}
 <p>
-  {{ pattern('button', {
-    label: label,
-    style: style,
-    outline: outline,
-    url: '#' ~ id,
-    size: size,
-    attributes: create_attribute({
-      'data-bs-toggle': 'collapse',
-      'role': 'button',
-      'aria-expanded': 'false',
-      'aria-controls': id
-    })
-  }) }}
+  {% if 'link' in variant %}
+    {{ pattern('link', {
+      label: label,
+      style: style,
+      variant: variant,
+      attributes: create_attribute({
+        'data-bs-toggle': 'collapse',
+        'role': 'button',
+        'aria-expanded': 'false',
+        'aria-controls': id
+      }),
+      path: '#' ~ id,
+    }) }}
+  {% else %}
+    {{ pattern('button', {
+      label: label,
+      style: style,
+      outline: outline,
+      variant: variant,
+      url: '#' ~ id,
+      size: size,
+      attributes: create_attribute({
+        'data-bs-toggle': 'collapse',
+        'role': 'button',
+        'aria-expanded': 'false',
+        'aria-controls': id
+      })
+    }) }}
+  {% endif %}
 </p>
 <div class="collapse" id="{{id}}">
   <div class="card card-body">
```

Comment:
normally we should use the card pattern, call it, set the body, no header no footer